### PR TITLE
Playlist v2

### PR DESCRIPTION
**Pull request by dipen2000**
Please review my playlist feature of the vid-library.

### CODE REVIEW

File: src/components/Cards/PlaylistCard/PlaylistCard.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      <div className="playlist-thumbnail-container bord-3-green">
+        {playlist.videos.length === 0 ? (
+          <div className="playlist-thumbnail-section">
+            <h3>Empty playlist</h3>
+          </div>
+        ) : (
+          <img
+            className="img-resp"
+            src={playlist.videos[0].img_src}
+            alt="thumbnail-img"
```
Comment (Shivani-1524) [inline src/components/Cards/PlaylistCard/PlaylistCard.js:20]: the alt attribute adds image/picture etc by default when read by a screen reader.
suggestion: video thumbnail instead

File: src/components/Cards/SinglePlaylistCard/SinglePlaylistCard.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+.video-thumbnail-container{
+    height : 15rem;
+}
+
+.video-details-container{
+    padding : 0.5rem;
+}
+
+.video-btn-CTA-container{
+    padding : 0.5rem;
```
Comment (Shivani-1524) [inline src/components/Cards/SinglePlaylistCard/SinglePlaylistCard.css:14]: for units lesser than 1 rem, pixels should do fine but this is ok too

File: src/components/Chip/Chip.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.chip{
+    padding : 0.7rem 1.5rem;
+    border : 3px solid black;
```
Comment (Shivani-1524) [inline src/components/Chip/Chip.css:3]: try using color variables everywhere even if its just black

**Review by Shivani-1524 — APPROVED** [review]
Great work

**Review by BALAJI141031 — APPROVED** [review]
checked files related to playlist.looking good

**Pull request merged**